test/iron-iconset-svg.html

```diff
@@ -65,6 +74,10 @@
         test('it can be accessed via iron-meta', function () {
           expect(meta.byKey('foo')).to.be.equal(iconset);
         });
+
+        test('it fires an iron-iconset-added event on the window', function() {
+          return loadedPromise;
```

Comment:
Nope! Mocha is awesome like that. If a test returns a promise then it must resolve within a timeout (10s by default) or the test fails.